app/helpers/application_helper.rb

```diff
@@ -0,0 +1,5 @@
+module ApplicationHelper
+  def classes(route, needed_classes='')
```

Comment:
This helper method would be globally available across all views so wonder if it'd be worth qualifying as being specific to "current page navigation" logic?

e.g.

```suggestion
  def classes_if_current_route(route, needed_classes='')
```

Another idea could be to take advantage of Rails tag helpers support for classes as an array, and filter dynamically.

e.g.

```html
<%= link_to 'Apps', service_providers_path, class: [current_page?(service_providers_path) && 'usa-current', 'usa-nav__link'] %>
```

Or maybe abstracting `link_to`

```suggestion
  def navigation_link_to(text, route)
    link_to text, route, class: [current_page?(route) && 'usa-current', 'usa-nav__link']
```

Reply:
hm definitely worth changing. 

i think i like the last option best -- was trying to get the logic out of the view, so i'll update!

Reply:
updated in https://github.com/18F/identity-dashboard/pull/750/commits/eb88d9f8a048032876c240f0196b7d00051eaa84